addons/l10n_in/views/res_config_settings_views.xml

```diff
@@ -10,62 +10,132 @@
-                                        type="object"
-                                        string="Buy credits"
-                                        class="btn-link"/>
+                        <setting name="l10n_in_tds"
+                                 help="Enable this to activate the Tax Deducted at Source (TDS) feature and the account-based TDS section suggestion."
+                                 string="TDS"
+                                 documentation="/applications/finance/fiscal_localizations/india.html">
+                            <field name="l10n_in_tds_feature"/>
+                            <div invisible="not l10n_in_tds_feature">
+                                <div class="row">
+                                    <label string="TAN" for="l10n_in_tan" class="col-2 o_light_label"/>
+                                    <field name="l10n_in_tan"/>
+                                </div>
                             </div>
                         </setting>
-                        <setting name="l10n_in_tds_tcs"
-                                 help="Enable this to activate Tax Deduction Source and Tax Collection Source."
-                                 string="TDS and TCS"
+                        <setting name="l10n_in_tcs"
+                                 help="Enable this to activate the Tax Collected at Source (TCS) feature and the account-based TCS section suggestion."
+                                 string="TCS"
                                  documentation="/applications/finance/fiscal_localizations/india.html">
-                            <field name="module_l10n_in_withholding"/>
-                        </setting>
-                        <setting name="electronic_invoices_in"
-                                 help="Connect to NIC (National Informatics Center) to submit invoices on posting."
-                                 company_dependent="1"
-                                 documentation="/applications/finance/fiscal_localizations/india.html#india-e-invoicing">
-                            <field name="module_l10n_in_edi"/>
-                        </setting>
-                        <setting name="electronic_waybill_in"
-                                 help="Connect to NIC (National Informatics Center) to submit e-waybill on posting."
-                                 company_dependent="1"
-                                 documentation="/applications/finance/fiscal_localizations/india.html#indian-e-waybill">
-                            <field name="module_l10n_in_ewaybill" class="oe_inline"/>
+                            <field name="l10n_in_tcs_feature"/>
+                            <div invisible="not l10n_in_tcs_feature">
+                                <div class="row">
+                                    <label string="TAN" for="l10n_in_tan" class="col-2 o_light_label"/>
+                                    <field name="l10n_in_tan"/>
```

Comment:
We can not put the TAN separately and make it visible if either TCS/TDS is selected?  It is weird to see 2 fields for the TAN when it is just the same field.

Reply:
okay